jplag/src/main/java/de/jplag/JPlagComparison.java

```diff
@@ -139,44 +68,15 @@ public final float similarityOfFirst() {
      */
     public final float similarityOfSecond() {
         int divisor = secondSubmission.getSimilarityDivisor(true);
-        return (divisor == 0 ? 0f : (getNumberOfMatchedTokens() * 100 / (float) divisor));
-    }
-
-    /**
-     * @return Similarity in percent rounded down to the nearest tenth.
-     */
-    public final float roundedSimilarity() {
-        return ((int) (similarity() * ROUNDING_FACTOR)) / (float) ROUNDING_FACTOR;
-    }
-
-    /**
-     * @return Similarity of the first submission to the basecode in percent rounded down to the nearest tenth.
-     */
-    public final float basecodeSimilarityOfFirst() {
-        return ((int) (firstBasecodeSimilarity() * ROUNDING_FACTOR)) / (float) ROUNDING_FACTOR;
-    }
-
-    /**
-     * @return Similarity of the second submission to the basecode in percent rounded down to the nearest tenth.
-     */
-    public final float basecodeSimilarityOfSecond() {
```

Comment:
Whats the reason for the removal of these methods?

Reply:
1) They are not needed
2) The values regarding base code can be accessed using the `Submission#getBaseCodeComparison#similarity`

Reply:
Sounds good to me. If the Artemis crowd complains (as these methods are part of the public interface), we can always adjust.